### DISCUSSION TITLE
GroupSpeedDial needs a search "search IN group"

### DISCUSSION THREAD

**Discussion by torstenzenk** [fastaddons/GroupSpeedDial#285] [Ideas]
Hi Juraj,

i love your group Speeddial addon and also support you via patreon. It is "almost" perfect.
The only thing that i definitly miss quite often is a search within my groups, as i have so much entries in certain groups and the previews are too small and i cant really see real good, that i would love to use this feature-idea:
1. directly give in a search word in the exisiting search window (where it says at the moment "SEARCH GROUPS")
2. from there it will not just open the group where an entry of the search-string is , but directly focusses the first found preview with some lines around that preview and colorizes the header in a yellow or red (or other color) background, where this search string exists in.
3. To make it perfect i can create a key-setting to perform a "find the next result" of that search-string (like a strg+g or so) if the first found result is not the right one.

Would that be possible and would it be makeable?
Best Regards
Torsten

**Comment by Juraj-Masiar** [comment]
Hello Torsten,
Sure thing! :)
The "search for groups" you see in the popup window (that's what you mean, right?) is very old, it was implemented in the very beginning about 5 years ago so it deserves some update :).

Here is what I'm thinking I can do:
- add new global hotkey that will open popup menu (in Chrome you can define one yourself by setting "Activate the extension" shortcut in the "chrome://extensions/shortcuts" page)
- rebuild the whole popup window using my Vue framework (this will allow me to use rich UI features with filters, options, highlights and more)
- make it fully usable with keyboard, so that in ideal case you could on any page press keyboard shortcut, write a few letters and press Enter. Ideally with some option to open all search results in multiple tabs. And with option to open the dials page with highlighted dial (if you are looking for a dial, not the target page).

Currently I'm finishing a big "Share dials" feature which should be finished in a week or two so I'll try to schedule this next.
Note however that my estimates are sometimes too optimistic so please be patient :)

**Reply by torstenzenk** [reply]
Hello Juraj,
yes that is what i mean :). Your plans sound good to me and don´t worry. :) i use your extension since years "as it is", so no need to rush . Thanks for picking this idea up. 
Best
Torsten

**Comment by torstenzenk** [comment]
Hi Juraj,
the search feature is great now. Thank you very much!
